modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,24 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. *Optional*: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
+Devfile::
+If your Git repository contains a Devfile it would be automatically detected.
+* The Devfile would be automatically detected and populated on the *Devfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
+* If multiple Devfiles are detected, the Devfile path would need to be specified, relative to the *Context Dir* field, to import a specific Devfile.
+
+Dockerfile::
+If your Git repository contains a Dockerfile it would be automatically detected.
+* The Dockerfile would be automatically detected and populated on the *Dockerfile path* field.
+* To edit the file import type, click *Edit import strategy* option, to select a different strategy.
+* If multiple Dockerfiles are detected, the Dockerfile path would need to be specified, relative to the *Context Dir* field, to import a specific Dockerfile.
+
+Devfile and Dockerfile::
+* If a Devfile and a Dockerfile is detected in the same repository, the Devfile is selected by default. You can click on the *Edit import strategy* to choose a different strategy.
+* If multiple Devfiles and Dockerfiles are detected in the same repository, you can specify which file you wish to import.
+
+. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. You can change the version using the *Builder Image Version* drop-down list. If required, you can use the *Edit import Strategy*, to select the desired strategy.
```

Comment:
It is unclear from the context, but it sounds like the following line describes the result of a previous user action. Maybe look for a better position for this information. It should not be a numbered step. Then, list the required and optional user actions. For example:

```
After <agent> validates the URL in the *Builder* section, it finds and selects an appropriate builder image and marks it with a star. For `https://github.com/sclorg/nodejs-ex`, <agent> selects the default Node.js builder image. 

. If <agent> does not auto-detect a builder image, select a builder image. 
. Optional: Use *Builder Image Version* to specify a version. 
. Optional: Use *Edit import Strategy* to select a strategy.
```